graph/pull_v2.go

```diff
@@ -368,6 +368,26 @@ func (p *v2Puller) verifyTrustedKeys(namespace string, keys []libtrust.PublicKey
 }
 
 func (p *v2Puller) validateManifest(m *manifest.SignedManifest, tag string) (verified bool, err error) {
+	// If pull by digest, then verify the manifest digest.
```

Comment:
@stevvooe I seem to remember us working on this code somewhere else in the codebase to implement hash verification of manifests. Is this redundant with that implementation?